buildSrc/subprojects/buildquality/src/main/kotlin/org/gradle/gradlebuild/buildquality/classycle/Classycle.kt

```diff
@@ -108,7 +96,7 @@ open class Classycle : DefaultTask() {
                 } catch (ex: Exception) {
                     try {
                         "unzip"(
-                            "src" to reportResourcesZip.asFile.get(),
+                            "src" to reportResourcesZip.get().asFile,
```

Comment:
Yeah, I don't quite understand the reason why we have both ways to get a file from a `RegularFileProperty`.